docs/static/v2.1.0-rc1/cloud-config.json

```diff
@@ -0,0 +1,415 @@
+{
+ "$schema": "https://kairos.io/v2.0.0-rc4/cloud-config.json",
```

Comment:
@Itxaka do we have the right version of the agent running against master? could be a different bug but rather ping you just to be sure

Reply:
Not yet! Just bumped the agent to final 2.0.0 and triggered the packages.

I mean, it makes sense in this case because it's using an rc version of the agent here, so it did it ok

Reply:
The only problem I can see its that this now reports the version of the kairos-agent package :/

I guess that its kind of okay, because the schema is provided by the agent? But yeah, does not link with the others that used the kairos version for the schema...


@mauromorales I guess we need to be able to generate the version for the kairos version, instead of the agent. Which is kind of ugly I guess as bumping kairos does not mean that the schema has changed. On the other side, bumping the agent also does not mean that the schema has changed....

Probably needs a small discussion on how to approach this.